src/js/views/patients/sidebar/action/action-sidebar-attachments-views.js

```diff
@@ -0,0 +1,57 @@
+import hbs from 'handlebars-inline-precompile';
+import { View, CollectionView } from 'marionette';
+
+import { alphaSort } from 'js/utils/sorting';
+
+import 'scss/modules/sidebar.scss';
+
+import './action-sidebar.scss';
+
+const AttachmentView = View.extend({
+  className: 'u-margin--t-16',
+  template: hbs`
+    <a class="action-sidebar__attachment-filename" target="_blank" href={{path}}>{{filename}}</a>
+    <div>
+      <a class="action-sidebar__attachment-download" href={{path}} download>
+        {{far "download"}} <span>{{ @intl.patients.sidebar.action.attachmentsViews.attachmentView.downloadText }}</span>
+      </a>
+    </div>
+  `,
+  templateContext() {
+    return {
+      filename: this.model.get('path').split('/').pop(),
+    };
+  },
+});
+
+const AttachmentsCollectionView = CollectionView.extend({
+  childView: AttachmentView,
+  viewComparator(viewA, viewB) {
+    return alphaSort('desc', viewA.model.get('created_at'), viewB.model.get('created_at'));
+  },
+});
+
+const AttachmentsView = View.extend({
```

Comment:
Same here this template can just move to the collectionview using `childViewContainer`

Reply:
Updated in the [last push](https://github.com/RoundingWell/care-ops-frontend/compare/94513d9123ff0fc3bb004d21722f66285224fbce..d40905e94fa7c8344e44f93bef6cc13d1097e014).